### PR TITLE
fix(procedures): Translate form title and button, add subtitle

### PR DESCRIPTION
**Pull request by cedricmessiant**

### CODE REVIEW

**Comment by y-lohse** [comment]
Hmm, I did similar work on the `Overview` page, but I chose to use the `Subtitle` and `Title` components instead of the utility classes. I think it's best if we write as little css as possible, and that includes utility classes. WDYT?

File: packages/cozy-procedures/src/components/PersonalDataForm.jsx [hunk 1]
```diff
@@ -25,10 +26,11 @@ const widgets = {
 
 class PersonalDataForm extends React.Component {
   render() {
-    const { formData, updateFormData, router } = this.props
+    const { formData, updateFormData, router, t } = this.props
     return (
       <div className="u-ml-2">
-        <h2 className="u-title-h2">Personal data</h2>
+        <h2 className="u-title-h2">{t('personalData.title')}</h2>
+        <h3 className="u-title-h3">{t('personalData.subtitle')}</h3>
```
Comment (ptbrowne) [inline packages/cozy-procedures/src/components/PersonalDataForm.jsx]: Why do we need to do that ? Aren't there typographic components in cozy-ui ?

**Comment by cedricmessiant** [comment]
> Hmm, I did similar work on the `Overview` page, but I chose to use the `Subtitle` and `Title` components instead of the utility classes. I think it's best if we write as little css as possible, and that includes utility classes. WDYT?

yeah, of course !

File: packages/cozy-procedures/src/components/PersonalDataForm.jsx (hunk 1 above)
Comment (cedricmessiant) [reply packages/cozy-procedures/src/components/PersonalDataForm.jsx]: fixed it after @y-lohse 's comment, thanks

**Pull request merged**